lib/netbox_client_ruby/api/dcim/inventory_item.rb

```diff
@@ -0,0 +1,15 @@
+require 'netbox_client_ruby/entity'
+require 'netbox_client_ruby/api/dcim/device'
+
+module NetboxClientRuby
+  class InventoryItem
+    include NetboxClientRuby::Entity
```

Comment:
I really don't understand why that `Entity` class has to be so complex...https://github.com/ninech/netbox-client-ruby/blob/master/lib/netbox_client_ruby/entity.rb